### ISSUE TITLE
`paint-order` not applied to `<use>`

### ISSUE THREAD

**Issue by stanio** [weisJ/jsvg#83]
<details>
<summary><code>use-paint-order.svg</code></summary>

```xml
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 240 240">
  <defs>
    <g id="a-circle">
      <circle r="80" />
    </g>
  </defs>

  <circle cx="90" cy="90" r="80" fill="green"
      stroke="blue" stroke-width="10" paint-order="stroke fill" />

  <use href="#a-circle" x="150" y="150" fill="orange"
      stroke="blue" stroke-width="10" paint-order="stroke fill" />

</svg>
```
</details>

Actual | Expected
--- | ---
![actual](https://github.com/weisJ/jsvg/assets/1247730/4bef2e23-68a2-41b3-be4b-5261783ab332) | ![expected](https://github.com/weisJ/jsvg/assets/1247730/9ab634c4-362e-488b-9c63-f87f862e5036)

**Comment by weisJ** [comment]
I'm not sure behaviour of other implementations actually is correct here. Per the spec https://svgwg.org/svg2-draft/painting.html#PaintOrder the `paint-order` property is only allowed on the following elements
- shapes:
  - circle
  - ellipse
  - line
  - path
  - polygon
  - poly line
  - rect
- text content elements 
  - text
  - textPath
  - span
 
It is inherited though that should only apply to text content elements as out of the above elements only `text` actually is a container type (in the sense that is can have visible children).

**Comment by stanio** [comment]
Looks similar to the [`fill`](https://svgwg.org/svg2-draft/painting.html#SpecifyingFillPaint) attribute.  It is said it applies to just shapes and text content elements but then it is inherited and one could specify `fill="..."` on the root `<svg>` or `<g>` element – that one is handled by JSVG.

**Comment by stanio** [comment]
I think _"applies to"_ and which element could specify the attribute are two different parts.  The `<use>` element is specified to have [presentation attributes](https://svgwg.org/svg2-draft/styling.html#TermPresentationAttribute) that include `paint-order`.  [_"Style Scoping and Inheritance"_](https://svgwg.org/svg2-draft/struct.html#UseStyleInheritance) further specify how these apply to referenced elements.

**Comment by weisJ** [comment]
Ah it has been to long since I worked on inherited properties. This makes a lot of sense.

**Comment by weisJ** [comment]
Should be fixed in current snapshot

**Comment by stanio** [comment]
Tested with current 1.5.1-SNAPSHOT – works as expected now.

**Comment by weisJ** [comment]
Now available in 1.6.0
